js/message_test.js

```diff
@@ -145,11 +145,7 @@ describe('Message test suite', function() {
         undefined, undefined, undefined, undefined]);
     var result = foo.toObject();
     assertObjectEquals({
-      aString: undefined,
-      anOutOfOrderBool: undefined,
-      aNestedMessage: {
-        anInt: undefined
-      },
+      aNestedMessage: {},
```

Comment:
I think toObject() should reflect the implicit defaults (0/""/false) just like the accessors would. I'm double-checking on this.